cves/CVE-2015-5145.yml

```diff
@@ -110,10 +122,13 @@ discovered:
     The automated, contest, and developer flags can be true, false, or nil.
 
     If there is no evidence as to how this vulnerability was found, then please explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    This vulnerability was reported to the Django team by a Joao Silva and 
+    Ross Brunton. They do not appear to be Django developers. Although they
```

Comment:
Just grammar suggestion, might be better to connect the two sentences? - Although they contribute to the github repos, the two developers do not appear to be Django developers.